cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/test/java/io/cdap/cdap/datapipeline/DraftServiceTest.java

```diff
@@ -365,12 +365,13 @@ private Draft createBatchPipelineDraft(DraftId draftId, String name, String desc
         .addConnection("src", "sink")
         .setEngine(Engine.SPARK)
         .build();
+    String parentVersion = "dummyParentVersion";
 
```

Comment:
can we also do a test when parentVersion is null?

Reply:
updated a null test